junit-jupiter-api/src/main/java/org/junit/jupiter/api/Assumptions.java

```diff
@@ -42,9 +42,11 @@
 @API(Maintained)
 public final class Assumptions {
 
+	///CLOVER:OFF
 	private Assumptions() {
 		/* no-op */
 	}
+	///CLOVER:ON
```

Comment:
Oops, too quick for my own good! I don't think I've dealt with this over at #1027 yet, so I'll deal with that as soon as I can.